diff --git a/api.js b/api.js
index 22742c7..cab2d8d 100644
--- a/api.js
+++ b/api.js
@@ -1,4 +1,4 @@
-const yelpApiKey = "Bearer fpfUJj8DFp_jm-n0LNi5U4WL9AgyD3G2ieoAPAYccY2QUi-1ZCXSuHoa0uEaPY60BInSS_COQHHlqWp0VeKDOcgdPBHn9lYSC1_r6mJCI3y8aU63IHNfK6Lhr3xhXXYx";
-const corsAnywhereUrl = "https://cors-anywhere.herokuapp.com/";
+const yelpApiKey = 'Bearer fpfUJj8DFp_jm-n0LNi5U4WL9AgyD3G2ieoAPAYccY2QUi-1ZCXSuHoa0uEaPY60BInSS_COQHHlqWp0VeKDOcgdPBHn9lYSC1_r6mJCI3y8aU63IHNfK6Lhr3xhXXYx'
+const corsAnywhereUrl = 'https://cors-anywhere.herokuapp.com/'
 L.mapbox.accessToken =
-"pk.eyJ1Ijoic3VlcGFyazA5IiwiYSI6ImNqenJmdGxoNzBqengzbW8zeDlmNnhudHEifQ.NvYx9iu9NUGdvDdYdWNg-A";
\ No newline at end of file
+'pk.eyJ1Ijoic3VlcGFyazA5IiwiYSI6ImNqenJmdGxoNzBqengzbW8zeDlmNnhudHEifQ.NvYx9iu9NUGdvDdYdWNg-A'
diff --git a/bideo.js b/bideo.js
index 6df5d4e..bcb71a3 100644
--- a/bideo.js
+++ b/bideo.js
@@ -6,13 +6,13 @@
  * - http://www.w3schools.com/tags/ref_av_dom.asp
  */
 
-(function(global) {
+(function (global) {
   // Define Bideo constructor on the global object
-  global.Bideo = function() {
+  global.Bideo = function () {
     // Plugin options
-    this.opt = null;
+    this.opt = null
     // The Video element
-    this.videoEl = null;
+    this.videoEl = null
 
     // Approximate Loading Rate
     //
@@ -23,142 +23,140 @@
     // decide whether to play the video or not.
     // This behaviour will be controller with
     // the `acceptableLoadingRate` option.
-    this.approxLoadingRate = null;
+    this.approxLoadingRate = null
 
     // Methods to which `this` will be bound
-    this._resize = null;
-    this._progress = null;
+    this._resize = null
+    this._progress = null
 
     // Time at which video is initialized
-    this.startTime = null;
+    this.startTime = null
 
-    this.onLoadCalled = false;
+    this.onLoadCalled = false
 
     // Initialize and setup the video in DOM`
-    this.init = function(opt) {
+    this.init = function (opt) {
       // If not set then set to an empty object
-      this.opt = opt = opt || {};
+      this.opt = opt = opt || {}
 
-      var self = this;
+      var self = this
 
-      self._resize = self.resize.bind(this);
+      self._resize = self.resize.bind(this)
 
       // Video element
-      self.videoEl = opt.videoEl;
+      self.videoEl = opt.videoEl
 
       // Meta data event
-      self.videoEl.addEventListener("loadedmetadata", self._resize, false);
+      self.videoEl.addEventListener('loadedmetadata', self._resize, false)
 
       // Fired when enough has been buffered to begin the video
       // self.videoEl.readyState === 4 (HAVE_ENOUGH_DATA)
-      self.videoEl.addEventListener("canplay", function() {
+      self.videoEl.addEventListener('canplay', function () {
         // Play the video when enough has been buffered
         if (!self.opt.isMobile) {
-          self.opt.onLoad && self.opt.onLoad();
-          if (self.opt.autoplay !== false) self.videoEl.play();
+          self.opt.onLoad && self.opt.onLoad()
+          if (self.opt.autoplay !== false) self.videoEl.play()
         }
-      });
+      })
 
       // If resizing is required (resize video as window/container resizes)
       if (self.opt.resize) {
-        global.addEventListener("resize", self._resize, false);
+        global.addEventListener('resize', self._resize, false)
       }
 
       // Start time of video initialization
-      this.startTime = new Date().getTime();
+      this.startTime = new Date().getTime()
 
       // Create `source` for video
-      this.opt.src.forEach(function(srcOb, i, arr) {
-        var key,
-          val,
-          source = document.createElement("source");
+      this.opt.src.forEach(function (srcOb, i, arr) {
+        var key
+        var val
+        var source = document.createElement('source')
 
         // Set all the attribute key=val supplied in `src` option
         for (key in srcOb) {
           if (srcOb.hasOwnProperty(key)) {
-            val = srcOb[key];
+            val = srcOb[key]
 
-            source.setAttribute(key, val);
+            source.setAttribute(key, val)
           }
         }
 
-        self.videoEl.appendChild(source);
-      });
+        self.videoEl.appendChild(source)
+      })
 
       if (self.opt.isMobile) {
         if (self.opt.playButton) {
-          self.opt.videoEl.addEventListener("timeupdate", function() {
+          self.opt.videoEl.addEventListener('timeupdate', function () {
             if (!self.onLoadCalled) {
-              self.opt.onLoad && self.opt.onLoad();
-              self.onLoadCalled = true;
+              self.opt.onLoad && self.opt.onLoad()
+              self.onLoadCalled = true
             }
-          });
+          })
 
           self.opt.playButton.addEventListener(
-            "click",
-            function() {
-              self.opt.pauseButton.style.display = "inline-block";
-              this.style.display = "none";
+            'click',
+            function () {
+              self.opt.pauseButton.style.display = 'inline-block'
+              this.style.display = 'none'
 
-              self.videoEl.play();
+              self.videoEl.play()
             },
             false
-          );
+          )
 
           self.opt.pauseButton.addEventListener(
-            "click",
-            function() {
-              this.style.display = "none";
-              self.opt.playButton.style.display = "inline-block";
+            'click',
+            function () {
+              this.style.display = 'none'
+              self.opt.playButton.style.display = 'inline-block'
 
-              self.videoEl.pause();
+              self.videoEl.pause()
             },
             false
-          );
+          )
         }
       }
-
-      return;
-    };
+    }
 
     // Called once video metadata is available
     //
     // Also called when window/container is resized
-    this.resize = function() {
+    this.resize = function () {
       // IE/Edge still don't support object-fit: cover
-      if ("object-fit" in document.body.style) return;
+      if ('object-fit' in document.body.style) return
 
       // Video's intrinsic dimensions
-      var w = this.videoEl.videoWidth,
-        h = this.videoEl.videoHeight;
+      var w = this.videoEl.videoWidth
+      var h = this.videoEl.videoHeight
 
       // Intrinsic ratio
       // Will be more than 1 if W > H and less if H > W
-      var videoRatio = (w / h).toFixed(2);
+      var videoRatio = (w / h).toFixed(2)
 
       // Get the container DOM element and its styles
       //
       // Also calculate the min dimensions required (this will be
       // the container dimentions)
-      var container = this.opt.container,
-        containerStyles = global.getComputedStyle(container),
-        minW = parseInt(containerStyles.getPropertyValue("width")),
-        minH = parseInt(containerStyles.getPropertyValue("height"));
+      var container = this.opt.container
+      var containerStyles = global.getComputedStyle(container)
+      var minW = parseInt(containerStyles.getPropertyValue('width'))
+      var minH = parseInt(containerStyles.getPropertyValue('height'))
 
       // If !border-box then add paddings to width and height
-      if (containerStyles.getPropertyValue("box-sizing") !== "border-box") {
-        var paddingTop = containerStyles.getPropertyValue("padding-top"),
-          paddingBottom = containerStyles.getPropertyValue("padding-bottom"),
-          paddingLeft = containerStyles.getPropertyValue("padding-left"),
-          paddingRight = containerStyles.getPropertyValue("padding-right");
-
-        paddingTop = parseInt(paddingTop);
-        paddingBottom = parseInt(paddingBottom);
-        paddingLeft = parseInt(paddingLeft);
-        paddingRight = parseInt(paddingRight);
-
-        minW += paddingLeft + paddingRight;
-        minH += paddingTop + paddingBottom;
+      if (containerStyles.getPropertyValue('box-sizing') !== 'border-box') {
+        var paddingTop = containerStyles.getPropertyValue('padding-top')
+        var paddingBottom = containerStyles.getPropertyValue('padding-bottom')
+        var paddingLeft = containerStyles.getPropertyValue('padding-left')
+        var paddingRight = containerStyles.getPropertyValue('padding-right')
+
+        paddingTop = parseInt(paddingTop)
+        paddingBottom = parseInt(paddingBottom)
+        paddingLeft = parseInt(paddingLeft)
+        paddingRight = parseInt(paddingRight)
+
+        minW += paddingLeft + paddingRight
+        minH += paddingTop + paddingBottom
       }
 
       // What's the min:intrinsic dimensions
@@ -174,21 +172,21 @@
       // scale 500 to 700 and then calculate what should be the
       // right width. If we scale 1000 to 1200 then the height
       // will become 600 proportionately.
-      var widthRatio = minW / w;
-      var heightRatio = minH / h;
+      var widthRatio = minW / w
+      var heightRatio = minH / h
 
       // Whichever ratio is more, the scaling
       // has to be done over that dimension
       if (widthRatio > heightRatio) {
-        var new_width = minW;
-        var new_height = Math.ceil(new_width / videoRatio);
+        var new_width = minW
+        var new_height = Math.ceil(new_width / videoRatio)
       } else {
-        var new_height = minH;
-        var new_width = Math.ceil(new_height * videoRatio);
+        var new_height = minH
+        var new_width = Math.ceil(new_height * videoRatio)
       }
 
-      this.videoEl.style.width = new_width + "px";
-      this.videoEl.style.height = new_height + "px";
-    };
-  };
-})(window);
+      this.videoEl.style.width = new_width + 'px'
+      this.videoEl.style.height = new_height + 'px'
+    }
+  }
+})(window)
diff --git a/buttonClick.js b/buttonClick.js
index d3f28a2..818a64c 100644
--- a/buttonClick.js
+++ b/buttonClick.js
@@ -1,54 +1,54 @@
-var wrapper = document.getElementById("mainWrapper");
-var button = document.getElementById("ok");
+var wrapper = document.getElementById('mainWrapper')
+var button = document.getElementById('ok')
 
-button.addEventListener("click", function() {
-  wrapper.classList.toggle("blur");
-});
+button.addEventListener('click', function () {
+  wrapper.classList.toggle('blur')
+})
 
-var button2 = document.getElementById("notok");
+var button2 = document.getElementById('notok')
 
-button2.addEventListener("stuff", function() {
+button2.addEventListener('stuff', function () {
   var randomJoke = $.ajax({
     jsonp: false,
-    url: "http://api.icndb.com/jokes/random",
-    success: function(data) {
-      var joke = data.value.joke;
+    url: 'http://api.icndb.com/jokes/random',
+    success: function (data) {
+      var joke = data.value.joke
       // console.log(data)
-      console.log(joke);
+      console.log(joke)
       wrapper.innerHTML = `<div class="d-flex justify-content-center                           align-items-center flex-column">
       <h1>You are not old enough!</h1>
       <h2>Please get older and enjoy this joke!</h2>
       <h3>Random Chuck Norris Joke:</h3>
       <p>${joke}</p>
-      </div>`;
+      </div>`
     }
-  });
-  wrapper.classList.toggle("blur");
-});
+  })
+  wrapper.classList.toggle('blur')
+})
 
-function getJokeFromResponse(data) {
-  var joke = data.value.joke;
-  return joke;
+function getJokeFromResponse (data) {
+  var joke = data.value.joke
+  return joke
 }
 
-function renderJoke(joke) {
-  console.log(joke, "this is inside the second then, chained promise");
+function renderJoke (joke) {
+  console.log(joke, 'this is inside the second then, chained promise')
   wrapper.innerHTML = `<div class="d-flex justify-content-center align-items-center flex-column">
       <h1>You are not old enough!</h1>
       <h2>Please get older and enjoy this joke!</h2>
       <h3>Random Chuck Norris Joke:</h3>
       <p>${joke}</p>
-      </div>`;
-  wrapper.classList.toggle("blur");
+      </div>`
+  wrapper.classList.toggle('blur')
 }
 
-function renderUnderPage() {
+function renderUnderPage () {
   var randomJoke = $.ajax({
     jsonp: false,
-    url: "http://api.icndb.com/jokes/random"
+    url: 'http://api.icndb.com/jokes/random'
   })
     .then(getJokeFromResponse)
-    .then(renderJoke);
+    .then(renderJoke)
 }
 
-button2.addEventListener("click", renderUnderPage);
+button2.addEventListener('click', renderUnderPage)
diff --git a/homepage.js b/homepage.js
index 934c4f9..9d6ec7c 100644
--- a/homepage.js
+++ b/homepage.js
@@ -1,12 +1,12 @@
-document.addEventListener("DOMContentLoaded", function() {
-  console.log("Hello! The DOM is ready.");
-  document.getElementById("search-bar").value = "";
-  document.getElementById("search-form").addEventListener("keyup", function(e) {
-    e.preventDefault();
-    console.log(e);
-    console.log(sessionStorage);
-    sessionStorage.clear();
-    var location = document.getElementById("search-bar").value;
-    sessionStorage.setItem("location", location);
-  });
-});
+document.addEventListener('DOMContentLoaded', function () {
+  console.log('Hello! The DOM is ready.')
+  document.getElementById('search-bar').value = ''
+  document.getElementById('search-form').addEventListener('keyup', function (e) {
+    e.preventDefault()
+    console.log(e)
+    console.log(sessionStorage)
+    sessionStorage.clear()
+    var location = document.getElementById('search-bar').value
+    sessionStorage.setItem('location', location)
+  })
+})
diff --git a/main.js b/main.js
index 6cacd8c..e3b012d 100644
--- a/main.js
+++ b/main.js
@@ -1,31 +1,31 @@
-(function() {
-  var bv = new Bideo();
+(function () {
+  var bv = new Bideo()
   bv.init({
     // Video element
-    videoEl: document.querySelector("#background_video"),
+    videoEl: document.querySelector('#background_video'),
 
     // Container element
-    container: document.querySelector("body"),
+    container: document.querySelector('body'),
 
     // Resize
     resize: true,
 
     // autoplay: false,
 
-    isMobile: window.matchMedia("(max-width: 768px)").matches,
+    isMobile: window.matchMedia('(max-width: 768px)').matches,
 
-    playButton: document.querySelector("#play"),
-    pauseButton: document.querySelector("#pause"),
+    playButton: document.querySelector('#play'),
+    pauseButton: document.querySelector('#pause'),
 
     // Array of objects containing the src and type
     // of different video formats to add
     src: [
       {
-        src: "http://www.anstudio.co/wp-content/uploads/2019/08/Beer-24916.mp4",
-        type: "video/mp4"
+        src: 'http://www.anstudio.co/wp-content/uploads/2019/08/Beer-24916.mp4',
+        type: 'video/mp4'
       },
       {
-        src: "night.webm",
+        src: 'night.webm',
         type: 'video/webm;codecs="vp8, vorbis"'
       }
     ]
@@ -34,5 +34,5 @@
     // onLoad: function () {
     //   document.querySelector('#video_cover').style.display = 'none';
     // }
-  });
-})();
+  })
+})()
diff --git a/navigation.js b/navigation.js
index b7483e2..f345088 100644
--- a/navigation.js
+++ b/navigation.js
@@ -1,23 +1,21 @@
 function yelpAjax () {
-  var location = document.getElementById("search-bar").value;
+  var location = document.getElementById('search-bar').value
   var yelpAjaxInfo = {
     url: `${corsAnywhereUrl}https://api.yelp.com/v3/businesses/search?term=${searchTerm}&location=${location}&limit=10`,
     headers: { Authorization: `${yelpApiKey}` }
-  };
+  }
   return $.ajax(yelpAjaxInfo)
 }
 
 function placeMapboxMarkers (yelpSearchResults) {
- 
-  const viewCoordinate = yelpSearchResults.businesses[0].coordinates;
+  const viewCoordinate = yelpSearchResults.businesses[0].coordinates
   map
     .setView([viewCoordinate.latitude, viewCoordinate.longitude], 13)
     .addLayer(L.mapbox.styleLayer('mapbox://styles/mapbox/light-v10'))
 
-
   for (let i = 0; i < yelpSearchResults.businesses.length; i++) {
-    const coordinate = yelpSearchResults.businesses[i].coordinates;
-    const singleBusiness = yelpSearchResults.businesses[i];
+    const coordinate = yelpSearchResults.businesses[i].coordinates
+    const singleBusiness = yelpSearchResults.businesses[i]
     const marker = L.marker([coordinate.latitude, coordinate.longitude], {
       title: singleBusiness.name
     }).addTo(map).bindPopup(`
@@ -26,65 +24,65 @@ function placeMapboxMarkers (yelpSearchResults) {
                       }" height="50px"></div>
                       <h4>${singleBusiness.name}</h4>
                       ${singleBusiness.location.display_address.join(
-                        "<br>"
+                        '<br>'
                       )}         
-                  `);
-    marker.on("mouseover", function(e) {
-      this.openPopup();
-    });
-    marker.on("mouseout", function(e) {
-      this.closePopup();
-    });
+                  `)
+    marker.on('mouseover', function (e) {
+      this.openPopup()
+    })
+    marker.on('mouseout', function (e) {
+      this.closePopup()
+    })
   }
 }
 
-function deleteMarkers() {
+function deleteMarkers () {
   if (markersArray.length !== 0) {
-    markersArray.forEach(function(marker) {
-      marker.remove();
-    });
+    markersArray.forEach(function (marker) {
+      marker.remove()
+    })
   }
 }
 
-function createYelpResultsHtml(yelpSearchResults) {
-  const businessHtml = yelpSearchResults.map(function(singleBusiness) {
-    function renderStarRating() {
-      let starRating = "";
+function createYelpResultsHtml (yelpSearchResults) {
+  const businessHtml = yelpSearchResults.map(function (singleBusiness) {
+    function renderStarRating () {
+      let starRating = ''
       if (singleBusiness.rating === 5) {
-        starRating = '<img src="img/regular_5.png">';
+        starRating = '<img src="img/regular_5.png">'
       } else if (singleBusiness.rating === 4.5) {
-        starRating = '<img src="img/regular_4_half.png">';
+        starRating = '<img src="img/regular_4_half.png">'
       } else if (singleBusiness.rating === 4) {
-        starRating = '<img src="img/regular_4.png">';
+        starRating = '<img src="img/regular_4.png">'
       } else if (singleBusiness.rating === 3.5) {
-        starRating = '<img src="img/regular_3_half.png">';
+        starRating = '<img src="img/regular_3_half.png">'
       } else if (singleBusiness.rating === 3) {
-        starRating = '<img src="img/regular_3.png">';
+        starRating = '<img src="img/regular_3.png">'
       } else if (singleBusiness.rating === 2.5) {
-        starRating = '<img src="img/regular_2_half.png">';
+        starRating = '<img src="img/regular_2_half.png">'
       } else if (singleBusiness.rating === 2) {
-        starRating = '<img src="img/regular_2.png">';
+        starRating = '<img src="img/regular_2.png">'
       } else if (singleBusiness.rating === 1.5) {
-        starRating = '<img src="img/regular_1_half.png">';
+        starRating = '<img src="img/regular_1_half.png">'
       } else if (singleBusiness.rating === 1) {
-        starRating = '<img src="img/regular_1.png">';
+        starRating = '<img src="img/regular_1.png">'
       } else {
-        starRating = '<img src="img/regular_0.png">';
+        starRating = '<img src="img/regular_0.png">'
       }
-      return starRating;
+      return starRating
     }
-    function renderPriceRange() {
-      let priceRange = "";
-      if (singleBusiness.price === "$$$$") {
-        priceRange = '<span class="badge badge-pill badge-success">$$$$</span>';
-      } else if (singleBusiness.price === "$$$") {
-        priceRange = '<span class="badge badge-pill badge-success">$$$</span>';
-      } else if (singleBusiness.price === "$$") {
-        priceRange = '<span class="badge badge-pill badge-success">$$</span>';
+    function renderPriceRange () {
+      let priceRange = ''
+      if (singleBusiness.price === '$$$$') {
+        priceRange = '<span class="badge badge-pill badge-success">$$$$</span>'
+      } else if (singleBusiness.price === '$$$') {
+        priceRange = '<span class="badge badge-pill badge-success">$$$</span>'
+      } else if (singleBusiness.price === '$$') {
+        priceRange = '<span class="badge badge-pill badge-success">$$</span>'
       } else {
-        priceRange = '<span class="badge badge-pill badge-success">$</span>';
+        priceRange = '<span class="badge badge-pill badge-success">$</span>'
       }
-      return priceRange;
+      return priceRange
     }
     return `          
         <div id='card' class='card mb-3'>
@@ -99,27 +97,27 @@ function createYelpResultsHtml(yelpSearchResults) {
                         }"><h5 class='card-title'>${singleBusiness.name}</h5></a>
                         <p id='price-review'>${renderPriceRange()} • ${renderStarRating()} • ${singleBusiness.review_count} reviews XXX</p>
                         <p>${singleBusiness.location.display_address.join(
-                          "<br>"
+                          '<br>'
                         )}
                     </div>
                 </div>
             </div>
         </div>
-        `;
-  });
-  return businessHtml.join("");
+        `
+  })
+  return businessHtml.join('')
 }
 
 function insertHtmlIntoContainer (yelpSearchResults) {
-  const bodyContainer = document.getElementById("output");
-  bodyContainer.innerHTML = "";
+  const bodyContainer = document.getElementById('output')
+  bodyContainer.innerHTML = ''
   placeMapboxMarkers(yelpSearchResults)
-  ajaxResponse = yelpSearchResults.businesses;
-  bodyContainer.innerHTML = createYelpResultsHtml(ajaxResponse);
+  ajaxResponse = yelpSearchResults.businesses
+  bodyContainer.innerHTML = createYelpResultsHtml(ajaxResponse)
 }
 
-document.getElementById("search-form").addEventListener("submit", function(e) {
-  e.preventDefault();
+document.getElementById('search-form').addEventListener('submit', function (e) {
+  e.preventDefault()
   yelpAjax()
-    .then(insertHtmlIntoContainer);
-});
\ No newline at end of file
+    .then(insertHtmlIntoContainer)
+})
diff --git a/results.js b/results.js
index da0a1a6..3c9a6c2 100644
--- a/results.js
+++ b/results.js
@@ -1,18 +1,18 @@
-let ajaxResponse = null;
-const searchTerm = "happy hour";
-const map = L.mapbox.map("map");
-let markersArray = [];
+const ajaxResponse = null
+const searchTerm = 'happy hour'
+const map = L.mapbox.map('map')
+const markersArray = []
 
-document.addEventListener("DOMContentLoaded", function() {
-  const location = sessionStorage.getItem("location");
+document.addEventListener('DOMContentLoaded', function () {
+  const location = sessionStorage.getItem('location')
   var yelpAjaxRequest = {
     url: `${corsAnywhereUrl}https://api.yelp.com/v3/businesses/search?term=${searchTerm}&location=${location}&limit=10`,
     headers: { Authorization: `${yelpApiKey}` }
-  };
+  }
   deleteMarkers()
   $.ajax(yelpAjaxRequest)
-    .then(insertHtmlIntoContainer);
-});
+    .then(insertHtmlIntoContainer)
+})
 
 // // Start of load more results function (loads 10 at a time)
 // const loadButton = document.getElementById("load-more");
@@ -38,4 +38,4 @@ document.addEventListener("DOMContentLoaded", function() {
 //     var moreResultsHtml = createYelpResultsHtml(response);
 //     return resultsContainer.append(moreResultsHtml);
 //   });
-// });
\ No newline at end of file
+// });
